cmd/erasure-object.go

```diff
@@ -593,6 +614,8 @@ func (er erasureObjects) getObjectFileInfo(ctx context.Context, bucket, object s
 		if metaArr[i].IsValid() && metaArr[i].ModTime.Equal(fi.ModTime) {
 			continue
 		}
+		metaArr[i] = FileInfo{}
```

Comment:
I guess same thing need to be done after line 611 as well

Reply:
This is not needed it's just defensive.